deploy/base/config.yaml

```
@@ -9,3 +9,4 @@ metadata:
 data:
   SPOdImagePullPolicy: Always
   EnableSelinux: "false"
+  EnableLogEnricher: "false"
```

What is the reason that we have to quote those booleans? (I think I asked this before at the change above)
```suggestion
  EnableLogEnricher: false
```

---

ConfigMaps don't support it. Has to be `map[string]string` :(